src/main/java/uk/ac/ebi/intact/app/internal/ui/panels/filters/ToggleFilterPanel.java

```diff
@@ -81,11 +81,12 @@ protected void updateFilterUI(BooleanFilter<T> filter) {
         label.setText(filter.description);
         setButtonsEnabled();
 
-        boolean isThereNegativeInteractions = ((EdgePositiveFilter) filter).areThereNegativeInteractions();
-        boolean isTherePositiveInteractions = ((EdgePositiveFilter) filter).areTherePositiveInteractions();
+        boolean isNegativeHidden = ((EdgePositiveFilter) filter).isNegativeHidden();
+        boolean isPositiveHidden = ((EdgePositiveFilter) filter).isPositiveHidden();
 
-        negativeButton.setSelected(isThereNegativeInteractions && !isTherePositiveInteractions);
-        positiveButton.setSelected(isTherePositiveInteractions);
+        bothButton.setSelected(!isNegativeHidden && !isPositiveHidden);
```

Comment:
The negative filter already has a logic to set the right value (positive only, negative only) by default based on the interactions. We had that logic duplicated here. Since we already have that logic on the filter, here we just need to check what the filter has set to show that on the panel.